plugins/seagate/seagate-nvme.c

```diff
@@ -1645,13 +1645,13 @@ static int vs_clr_pcie_correctable_errs(int argc, char **argv, struct command *c
 {
 	const char *desc = "Clear Seagate PCIe Correctable counters for the given device ";
 	const char *save = "specifies that the controller shall save the attribute";
-<<<<<<< HEAD
```

Comment:
We don't want to have this in our git history. This looks like a left over from the development process.

Reply:
Only clean way I can see is to discard this pull request, create a new fork and do a new pull request. Guess I have to do that.